cdap-ui/app/features/operations/ops-ctrl.js

```diff
@@ -10,32 +10,22 @@ angular.module(PKG.name+'.feature.dashboard')
     var panels = [
    // Format:
    // [ Widget Title, context, [metricNames], line-type (options are in addwdgt-ctrl.js ]
-      ['Collect', '', ['system.collect.events'],           'c3-line'],
-      ['Process', '', ['system.process.events.processed'], 'c3-line'],
-      ['Store',   '', ['system.dataset.store.bytes'],      'c3-line'],
-      ['Query',   '', ['system.requests.count'],           'c3-line']
+      ['Router requests', '', ['system.request.received','system.response.client-error', 'system.response.successful'], 'c3-scatter'],
```

Comment:
Instead of 10 widgets on one dashboard, should we organise them into 2 different dashboards?
Otherwise, with 2 columns, that's 5 rows of widgets to scroll down (and scrolling can be annoying because it can also cause a chart to zoom in / zoom out if the mouse is over the chart when scrolling).